WireDomain/Sources/WireDomain/Synchronization/IncrementalSync.swift

```diff
@@ -114,4 +141,15 @@ public struct IncrementalSync {
         }
     }
 
+    public struct Token {
```

Comment:
todo: add description of what it does or what it is for?

Reply:
done c7b199f17f20d3c933cd32a31d23dc0ffa9a9d95